src/status_im/ui/components/react.cljs

```diff
@@ -36,6 +36,7 @@
 (def geolocation (when (exists? js/window)
                    js/navigator.geolocation.))
 (def view (get-class "View"))
+(def safe-area-view (get-class "SafeAreaView"))
```

Comment:
Is this always available? What happens if it is not?

Reply:
Yes, it is always available. React Native ships with `SafeAreaView` as of version 0.50.